specs/exec-engine.md

```diff
@@ -106,6 +107,10 @@ if there is an error processing the transactions. It must return `STATUS_VALID`
 be executed without error. **Note**: The state transition rules have been modified such that deposits will never fail
 so if `engine_forkchoiceUpdatedV1` returns `STATUS_INVALID` it is because a batched transaction is invalid.
 
+The `gasLimit` is optional w.r.t. compatibility with L1, but required when used as rollup.
```

Comment:
Could we add test coverage of this?

Reply:
Figured out how/where to add the test. Bindings do support the nil gas limit value, thought they did not previously.